### PR TITLE
Validate safeAppId for SafeMessage creation

### PR DESCRIPTION
**Pull request by Uxio0**

### CODE REVIEW

File: safe_transaction_service/safe_messages/serializers.py [hunk 1]
```diff
@@ -62,7 +69,9 @@ def get_valid_owner_from_signatures(
 
 class SafeMessageSerializer(SafeMessageSignatureParserMixin, serializers.Serializer):
     message = serializers.JSONField()
-    safe_app_id = serializers.IntegerField(allow_null=True, default=None)
+    safe_app_id = serializers.IntegerField(
+        allow_null=True, default=None, validators=[positive_integer_validator]
+    )
```
Comment (fmrsabino) [inline safe_transaction_service/safe_messages/serializers.py:74]: Maybe we can use `min_value=0` instead of a custom validator.

Wdyt?

Comment (Uxio0) [reply safe_transaction_service/safe_messages/serializers.py:74]: I think I'm still on holidays mode 🤦🏼‍♂️

Comment (Uxio0) [reply safe_transaction_service/safe_messages/serializers.py]: Updated

File: safe_transaction_service/safe_messages/tests/test_views.py [hunk 2]
```diff
@@ -175,13 +175,33 @@ def test_safe_messages_create_view(self, get_owners_mock: MagicMock):
...
                     format="json",
                     data=data,
                 )
                 self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+                self.assertEqual(
+                    response.data,
+                    {
+                        "safe_app_id": [
+                            ErrorDetail(
+                                string="A positive integer value must be greater than 0",
```
Comment (fmrsabino) [inline safe_transaction_service/safe_messages/tests/test_views.py:191]: This message should also be updated right?

**Review by fmrsabino — APPROVED** [review]

File: safe_transaction_service/safe_messages/tests/test_views.py (hunk 2 above)
Comment (Uxio0) [reply safe_transaction_service/safe_messages/tests/test_views.py]: Yes, done too!

**Pull request merged**